website/components/OvalLink.tsx

```diff
@@ -0,0 +1,20 @@
+import Link from '@docusaurus/Link';
+import classNames from 'classnames';
+import React from 'react';
+
+import styles from '../index.module.css';
+
+export type OvalLinkProps = JSX.IntrinsicElements['span'] & { to: string };
+
+export function OvalLink({ children, className, to, ...props }: OvalLinkProps) {
+  return (
+    <Link className={styles['oval-link']} to={to}>
+      <span
+        {...props}
+        className={classNames(styles['oval-link-icon'], className)}
```

Comment:
This class suggests that it's always going to be an icon. If so, I'd rather have an `IconLink` component.

Also, is it really that different from the others we have? Maybe we could _slightly_ extend an existing one instead?

Reply:
I stayed with a separate component. But that's right, we could use the oval component, expand it with a props link.